Official codes for ST16

EerieCode opened the pull request.

EerieCode committed:
Official codes for ST16

EerieCode committed:
Fixed "Magician Girl" setcodes

EerieCode committed:
Fixed "Red Dragon Archfiend Tyrant", "Odd-Eyes Mirage Dragon" and "Number 100: Numeron Dragon"

EerieCode committed:
TCG names for ST16

EerieCode committed:
Fixed OCG&TCG status, fixed "Dark Burning Attack", TCG names for WIRA and SHVI prerelease

EerieCode committed:
Fixed "Nirvana High Paladin"

EerieCode committed:
Fixed "Number 100: Numeron Dragon"

Steeldarkeagel commented:
Was going to merge this, but Git is having issues with your fixot.cdb file. Can you have a quick look into your file Eerie? What did you change with it?

EerieCode commented:
I added the cards that are still treated as OCG-only in it, and I've also
changed those cards and all WIRA cards to their TCG name and effects. If
it's a problem, wouldn't it be possible to reject only that commit and add
the others? If so, I'll try to reupload them later making sure there are no
conflicts.

2016-03-19 11:15 GMT+01:00 Steeldarkeagel notifications@github.com:

> Was going to merge this, but Git is having issues with your fixot.cdb
> file. Can you have a quick look into your file Eerie? What did you change
> with it?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/Ygoproco/Live/pull/189#issuecomment-198680174

EerieCode committed:
Revert "Fixed OCG&TCG status, fixed "Dark Burning Attack", TCG names for WIRA and SHVI prerelease"

This reverts commit d6315fb62ebc6941096e79e7de2e2ad20ae8433a.

EerieCode commented:
Ok, now it should be alright. I'll make a separate pull later.
